libs/core/src/lib/busy-indicator/busy-indicator-extended/busy-indicator-extended.directive.spec.ts

```diff
@@ -0,0 +1,76 @@
+import { ComponentFixture, TestBed, waitForAsync } from '@angular/core/testing';
+import { Component, TemplateRef, ViewChild, NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { BrowserModule } from '@angular/platform-browser';
+
+import { BusyIndicatorExtendedDirective } from './busy-indicator-extended.directive';
+import { MessageToastService } from '../../message-toast/message-toast-service/message-toast.service';
+import { MessageToastConfig } from '../../message-toast/message-toast-utils/message-toast-config';
+import { DynamicComponentService } from '@fundamental-ngx/core/utils';
+import { MessageToastComponent } from '../../message-toast/message-toast.component';
+import { MessageToastContainerComponent } from '../../message-toast/message-toast-utils/message-toast-container.component';
+import { BusyIndicatorComponent } from '../busy-indicator.component';
+
+@Component({
+    template: `<div #testTemplate fd-busy-indicator-extended [extended]="true">
+        <fd-busy-indicator [loading]="true" labelValue="Please wait" ariaLabel="Please wait"></fd-busy-indicator>
+    </div>`
+})
+class TestComponent {
+    @ViewChild('testTemplate', { static: true }) templateRef: TemplateRef<any>;
+}
+
+@NgModule({
+    declarations: [
+        MessageToastComponent,
+        MessageToastContainerComponent,
+        BusyIndicatorComponent,
+        BusyIndicatorExtendedDirective,
+        TestComponent
+    ],
+    imports: [CommonModule, BrowserModule],
+    providers: [MessageToastService, DynamicComponentService],
+    entryComponents: [
+        MessageToastComponent,
+        MessageToastContainerComponent,
+        BusyIndicatorComponent,
+        BusyIndicatorExtendedDirective,
+        TestComponent
+    ]
+})
+class TestModule {}
+
+describe('BusyIndicatorExtenedDirective', () => {
+    let component: MessageToastComponent;
+    let fixture: ComponentFixture<MessageToastComponent>;
+    let service: MessageToastService;
+
+    beforeEach(
+        waitForAsync(() => {
+            TestBed.configureTestingModule({
+                imports: [TestModule]
+            }).compileComponents();
+        })
+    );
+
+    beforeEach(() => {
+        fixture = TestBed.createComponent(MessageToastComponent);
+        component = fixture.componentInstance;
+        service = TestBed.get(MessageToastService);
+        fixture.detectChanges();
+    });
+
+    it('should create', () => {
+        expect(component).toBeTruthy();
+    });
+
+    it('should assign classes', () => {
+        service.open(TestComponent, { duration: 10 } as MessageToastConfig);
+        service['_messageToasts'][0].instance.ngOnInit();
+        fixture.detectChanges();
+        service['_messageToasts'][0].instance.ngAfterViewInit();
+        fixture.detectChanges();
+        expect(service.hasOpenMessageToasts()).toBe(true);
+        expect(fixture.nativeElement.classList.contains('fd-message-toast')).toBe(true);
```

Comment:
I don't see a test for `fd-busy-indicator-extended fd-busy-indicator-extended--message-toast` classes. This is what this directive is adding, isn't it?